services/verification/src/services/Injector.ts

```diff
@@ -561,7 +562,6 @@ export class Injector {
 
   public async storeMatch(
     chain: string,
-    addresses: string[],
     contract: CheckedContract,
     compilationResult: RecompilationResult,
     match: Match
```

Comment:
Not about this PR but it seemed weird to have an `address` in `Match` but not `chain`. WDYT?

Reply:
Indeed, maybe we solve this on staging once this PR is merged?

Reply:
Created https://github.com/ethereum/sourcify/issues/834  to keep track of this